diff --git src/kgx-pages.c src/kgx-pages.c
index c4dd662..ffa03ed 100644
--- a/src/kgx-pages.c
+++ b/src/kgx-pages.c
@@ -414,9 +414,9 @@ close_page (AdwTabView *view,
 
   children = kgx_tab_get_children (KGX_TAB (adw_tab_page_get_child (page)));
 
-  if (children->len < 1) {
+//  if (children->len < 1) {
     return GDK_EVENT_PROPAGATE; /* Aka no, I don’t want to block closing */
-  }
+//  }
 
   root = gtk_widget_get_root (GTK_WIDGET (self));
 
diff --git src/kgx-window.c src/kgx-window.c
index 15919fa..9d440d5 100644
--- a/src/kgx-window.c
+++ b/src/kgx-window.c
@@ -148,7 +148,7 @@ kgx_window_close_request (GtkWindow *window)
 
   children = kgx_pages_get_children (KGX_PAGES (priv->pages));
 
-  if (children->len < 1 || priv->close_anyway) {
+//  if (children->len < 1 || priv->close_anyway) {
     if (gtk_window_is_active (GTK_WINDOW (self))) {
       int width, height;
       gtk_window_get_default_size (GTK_WINDOW (self), &width, &height);
@@ -156,7 +156,7 @@ kgx_window_close_request (GtkWindow *window)
     }
 
     return FALSE; /* Aka no, I don’t want to block closing */
-  }
+//  }
 
   dlg = kgx_close_dialog_new (KGX_CONTEXT_WINDOW, children);
 
